shell.nix

```diff
@@ -1,19 +0,0 @@
-{ pkgs ? import (import ./nix/sources.nix {}).nixpkgs {} }:
```

Comment:
This removal should be followed by the update of `.envrc` to `use flake`. We could also remove it from the tracked files and add it to the ignored ones.